### PR TITLE
[Frontend][PaddlePaddle] Add topk op and Fix bug, when the output is a dimension, it …

### PR DESCRIPTION
**Pull request by woobinw**
There are some operation in PaddlePaddle that output a tensor which shape is [1], meanwhile, the slice op have decrease_axis attr, but it should not be squeezed.

@heliqi 
@jiangjiajun 
cc @junrushao

### CODE REVIEW

**Review by heliqi — APPROVED** [review]
LGTM

**Review by junrushao — APPROVED** [review]

File: tests/python/frontend/paddlepaddle/test_forward.py [hunk 1]
```diff
@@ -1681,5 +1687,15 @@ def forward(self, inputs, prev_h):
...
 
 
+@tvm.testing.uses_gpu
+def test_forward_topk():
+    @paddle.jit.to_static
+    def topk1(inputs):
+        return paddle.topk(inputs, k=1)
+
+    input_data = paddle.to_tensor([1, 4, 5, 7])
+    verify_model(topk1, input_data=input_data)
```
Comment (heliqi) [inline tests/python/frontend/paddlepaddle/test_forward.py:1697]: You can add a few different cases, example:
1. k is a tensor and not a int
2. set largest/axis/sort attribute 
3. The output is only the values or the indices.

Comment (woobinw) [reply tests/python/frontend/paddlepaddle/test_forward.py:1697]: according to paddle.topk api, there seems is no a argument to control the output that only have the values or indices

**Review by heliqi — APPROVED** [review]

File: tests/python/frontend/paddlepaddle/test_forward.py (hunk 1 above)
Comment (heliqi) [reply tests/python/frontend/paddlepaddle/test_forward.py:1718]: Yes, the paddle.topk api will definitely have two outputs.

**Review by jiangjiajun — APPROVED** [review]
LGTM

**Comment by junrushao** [comment]
Would you like to fix the CI so we could get it in?

**Comment by woobinw** [comment]
> Would you like to fix the CI so we could get it in?

I think 

> h

**Comment by woobinw** [comment]
> Would you like to fix the CI so we could get it in?
`E       File "/workspace/tests/python/frontend/paddlepaddle/test_forward.py", line 1356, in slice5 (* user code *)
[2023-01-05T11:12:37.644Z] E   	b, c, h, w = paddle.shape(inputs)  # add decrease_axis`
I think this error may be related to the version of the paddlepaddle

**Comment by heliqi** [comment]
> > Would you like to fix the CI so we could get it in?
> > `E       File "/workspace/tests/python/frontend/paddlepaddle/test_forward.py", line 1356, in slice5 (* user code *) [2023-01-05T11:12:37.644Z] E   	b, c, h, w = paddle.shape(inputs)  # add decrease_axis`
> > I think this error may be related to the version of the paddlepaddle

The paddlepaddle version we matched before was 2.1.3.
If you want to upgrade and match the latest paddlepaddle version, you can pull request and modify the Paddle version in [the code](https://github.com/apache/tvm/blob/a047e0228a3f7015e56c6756cdadb13444008623/docker/python/ci-constraints.txt
)  and [the code](https://github.com/apache/tvm/blob/98eab45d52911662ff760fb0bbdccc11f3a31b9f/docker/install/ubuntu_install_paddle.sh).

**Comment by woobinw** [comment]
> > > Would you like to fix the CI so we could get it in?
> > > `E       File "/workspace/tests/python/frontend/paddlepaddle/test_forward.py", line 1356, in slice5 (* user code *) [2023-01-05T11:12:37.644Z] E   	b, c, h, w = paddle.shape(inputs)  # add decrease_axis`
> > > I think this error may be related to the version of the paddlepaddle
> > > @woobinw
> > > The paddlepaddle version we matched before was 2.1.3.
> > > If you want to upgrade and match the latest paddlepaddle version 2.4, you can pull request and modify the Paddle version in [the code](https://github.com/apache/tvm/blob/a047e0228a3f7015e56c6756cdadb13444008623/docker/python/ci-constraints.txt)  and [the code](https://github.com/apache/tvm/blob/98eab45d52911662ff760fb0bbdccc11f3a31b9f/docker/install/ubuntu_install_paddle.sh).

`[2023-01-09T07:38:22.462Z] tests/python/frontend/paddlepaddle/test_forward.py:81: in verify_model

[2023-01-09T07:38:22.462Z]     baseline_outputs = baseline_model(*[input[:] for input in input_data])

[2023-01-09T07:38:22.462Z] /venv/apache-tvm-py3.7/lib/python3.7/site-packages/paddle/fluid/dygraph/layers.py:902: in __call__

[2023-01-09T07:38:22.462Z]     outputs = self.forward(*inputs, **kwargs)

[2023-01-09T07:38:22.462Z] /venv/apache-tvm-py3.7/lib/python3.7/site-packages/paddle/fluid/dygraph/io.py:1191: in __i_m_p_l__

[2023-01-09T07:38:22.462Z]     return _run_dygraph(self, input, program_holder)

[2023-01-09T07:38:22.462Z] /venv/apache-tvm-py3.7/lib/python3.7/site-packages/paddle/fluid/dygraph/io.py:754: in _run_dygraph`

**Comment by woobinw** [comment]
@junrushao 
The ethosu test seems to have nothing to do with PaddlePaddle frontend.

**Comment by junrushao** [comment]
Thank you for pinging me @woobinw! Indeed the failure is not relevant to the change you brought in, and I'm not sure if it's flaky or consistently reproducible error. Retriggered the test just now, and let's see if it could work this timw

**Comment by junrushao** [comment]
@tvm-bot rerun

**Comment by junrushao** [comment]
Could you rebase to latest HEAD? I’m sure it’s going to pass this time

**Comment by woobinw** [comment]
> Could you rebase to latest HEAD? I’m sure it’s going to pass this time

Yes, It has been rebased to latest HEAD.

**Comment by junrushao** [comment]
Thanks for the contribution! It’s merged!

**Pull request merged**